apps/dotcom/sync-worker/src/TLDrawDurableObject.ts

```diff
@@ -342,6 +343,17 @@ export class TLDrawDurableObject extends DurableObject {
 				if (!auth && !file.shared) {
 					return closeSocket(TLSyncErrorCloseEventReason.NOT_AUTHENTICATED)
 				}
+				if (auth?.userId) {
+					const rateLimited = await isRateLimited(this.env, auth?.userId)
+					if (rateLimited) {
+						return closeSocket(TLSyncErrorCloseEventReason.FORBIDDEN)
+					}
+				} else {
+					const rateLimited = await isRateLimited(this.env, sessionId)
```

Comment:
When would the sessionId default to instanceId. Could that cause some issues?

Reply:
don't think so